src/components/picker/helpers/usePickerSearch.tsx

```diff
@@ -23,13 +23,25 @@ const usePickerSearch = (props: UsePickerSearchProps) => {
     return children;
   }, [showSearch, searchValue, children]);
 
+  const filteredItems = useMemo(() => {
+    if (showSearch && !_.isEmpty(searchValue)) {
+      return _.filter(items, item => {
+        const {label, value} = item;
+        const itemLabel = getItemLabelPresenter(label, value);
+        return !shouldFilterOut(searchValue, itemLabel);
+      });
+    }
+
+    return items;
+  }, [showSearch, searchValue, items]);
```

Comment:
Can you reduce this code duplication?
IMO there should be a single function that uses both `children` and `items`